### DISCUSSION TITLE
query about color support

### DISCUSSION THREAD

**Discussion by GmBodhi** [markedjs/marked#2193] [Q&A] [answered]
Is it possible for me to use color syntax?

**Comment by UziTech** [comment, marked as answer]
Markdown doesn't have color support in the spec so we won't be adding it to marked. Their is a way to [extend](https://marked.js.org/using_pro#extensions) marked to support things like that but I don't know of any extensions that currently implement it.

**Reply by GmBodhi** [reply]
Okay, I might use `<span>`. Thank you for your time.
